modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,106 @@
+#include<iostream>
+#include<vector>
+#include<fstream>
+#include "opencv2/face/face_alignment.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+using namespace std;
+using namespace cv;
+int main(int argc,char** argv){
+    //Give the file containing the filenames of filenames as argument
+    if(argc!=2){
+        cout<<"No file found";
```

Comment:
actually, since you are checking here if the input variables are actually correct, you should use in my opining `cerr` instead of `cout`, because it is an actual error. 

Also make your message more meaningful, like `File containing filenames is not found, please provide it as an argument to the example + example command`.